qoqo_for_braket_devices/Cargo.toml

```diff
@@ -18,9 +18,9 @@ doctest = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-qoqo = { version = "1.9", default-features=false}
-roqoqo = "1.9"
-qoqo-macros = "1.9"
+qoqo = { version = "~1.11", default-features=false}
```

Comment:
Can we use 1.14 which is the latest release?

Reply:
Nope, the pyo3 update has not been merged in in here yet. `1.11` is the latest pyo3 `0.20` that there is.